src/rollback.rs

```diff
@@ -8,7 +8,7 @@ use bevy::{
 ///
 /// You must use the `AddRollbackCommand` when spawning an entity to add this component. Alternatively,
 /// you can use the `add_rollback()` extension method provided by `AddRollbackCommandExtension`.
-#[derive(Component, Hash, PartialEq, Eq, Clone, Copy, Debug, PartialOrd, Ord)]
+#[derive(Component, Hash, PartialEq, Eq, Clone, Copy, Debug)]
 pub struct Rollback(Entity);
```

Comment:
Aaaah. I totally forgot about https://github.com/gschup/bevy_ggrs/pull/58, and thought we we're still using "rollback ids"